makefile

```diff
@@ -7,22 +7,27 @@ BUILD=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
 BuildEnv="prod" 
 
 LDFLAGS=-ldflags "-w -s -X main.Version=${VERSION} -X main.BuildDate=${BUILD} -X main.BuildEnv=${BuildEnv}"
+# LDFLAGSWIN adds the -H=windowsgui flag to windows build to prevent cli from starting alongside godcr
+LDFLAGSWIN= -ldflags "-H=windowsgui -w -s -X main.Version=${VERSION} -X main.BuildDate=${BUILD} -X main.BuildEnv=${BuildEnv}"
 
-export GOARCH=amd64
-
-all: clean darwin windows
+all: clean macos windows linux freebsd
 
 freebsd:
-	GOOS=freebsd go build -trimpath ${LDFLAGS} -o ${BINARY}-freebsd-${GOARCH}
+	GOOS=freebsd GOARCH=amd64 go build -trimpath ${LDFLAGS} -o ${BINARY}-freebsd-${GOARCH}
+	GOOS=freebsd GOARCH=arm64 go build -trimpath ${LDFLAGS} -o ${BINARY}-freebsd-${GOARCH}
+
+linux:
+	GOOS=linux GOARCH=amd64 go build -trimpath ${LDFLAGS} -o ${BINARY}-linux-${GOARCH}
+	GOOS=linux GOARCH=arm64 go build -trimpath ${LDFLAGS} -o ${BINARY}-linux-${GOARCH}
```

Comment:
just out of curiosity, by do we have similar lines of codes repeated for all OS's? 
```
GOOS=freebsd GOARCH=amd64 go build -trimpath ${LDFLAGS} -o ${BINARY}-freebsd-${GOARCH}
GOOS=freebsd GOARCH=arm64 go build -trimpath ${LDFLAGS} -o ${BINARY}-freebsd-${GOARCH}
```
They are one and the same.

Reply:
Yeah
The only change is the GOOS which is the OS we are building for and the GOARCH which is the architecture we are building for.

Every other things is constant except for windows with a different LDFLAG